src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -6547,21 +6550,23 @@ public void onEviction(CityHash128Key key, PreparedStatementHandle handle) {
     /**
      * Checks if connection is established to SQL Azure server
      * 
-     * SERVERPROPERTY('EngineEdition') is used to determine if the db server is SQL Azure. It should return 6 for SQL
-     * Azure DW. This is more reliable than @@version or serverproperty('edition').
+     * SERVERPROPERTY('EngineEdition') is used to determine if the db server is SQL Azure. This is more reliable
+     * than @@version or serverproperty('edition').
      * 
-     * Reference: http://msdn.microsoft.com/en-us/library/ee336261.aspx
+     * Reference: https://docs.microsoft.com/en-us/sql/t-sql/functions/serverproperty-transact-sql?view=sql-server-ver15
```

Comment:
Is this specific to SQL Server 2019? If not, it's better to use the general URL and it will automatically redirect to any valid server version:
`https://docs.microsoft.com/en-us/sql/t-sql/functions/serverproperty-transact-sql`